chapters/synchronous.tex

```diff
@@ -1025,23 +994,13 @@ \subsection{Sub-clock Partitioning}\label{sub-clock-partitioning}
 
 \subsection{Sub-clock Inferencing}\label{sub-clock-inferencing}
 
-For each base-clock partition, the base interval needs to be determined
-and for each sub-clock partition, the sub-sampling factors and shift
-need to be determined. For each sub-clock partition, the interval might
-be rational or Real type and known or parametric or being unspecified.
-The sub-clock partition intervals are constrained by subSample and
-superSample factors which might be known (or parametric) or unspecified
-and by shiftSample shiftCounter and resolution or backSample,
-backCounter and resolution. This constraint set is used to solve for all
-intervals and sub-sampling factors and shift of the sub-clock
-partitions. The model is erroneous if no solution exist.
+For each base-clock partition, the base interval needs to be determined and for each sub-clock partition, the sub-sampling factors and shift need to be determined.  For each sub-clock partition, the interval might be rational or \lstinline!Real! type and known or parametric or being unspecified.  The sub-clock partition intervals are constrained by \lstinline!subSample! and \lstinline!superSample! factors which might be known (or parametric) or unspecified and by \lstinline!shiftSample! \lstinline!shiftCounter! and resolution or \lstinline!backSample!, \lstinline!backCounter! and resolution.  This constraint set is used to solve for all intervals and sub-sampling factors and shift of the sub-clock partitions.  The model is erroneous if no solution exist.
```

Comment:
Both shiftCounter and resolution are arguments to shiftSample, and backCounter and resolution are arguments of backSample. Thus it doesn't make sense to have syntax-highlighting for shiftCounter but not for resolution.
It was messed up before as well.

Reply:
I fixed the styling according to the rule that things referred by a named argument should use `\lstinline`.

Then, I found the sentence almost impossible to parse, not made easier by a missed comma.  I tried to improve this a little bit by tweaking the use of commas and a _as well as_ used as a low (operator) precedence _and_.  How about splitting the sentence in two?

Reply:
I think it needs to be rewritten more completely - but as separate PR.

Reply:
The proposed change was made, so it's fine for this one.